.rubocop.yml

```diff
@@ -158,8 +162,7 @@ Lint/EmptyInPattern: # new in 1.16
   Enabled: true
 
 Lint/IncompatibleIoSelectWithFiberScheduler: # new in 1.21
-  Enabled: false
-  # we don't use IO methods
+  Enabled: true
```

Comment:
Is there a reason you enabled this and the other one disabled for IO reasons below?

Reply:
If we do start using them, it would be good to make sure we follow best practices! I think they're things we want to follow if we do ever start doing IO in the app

Reply:
I did flag them with the reason for that reason! I mostly disabled them so that we'd be... running less linters... in the absence of that.

Reply:
discussed in discord, disabled!